src/scripts/main.js

```diff
@@ -1,3 +1,66 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const rows = table.rows;
+
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+const minCells = 2;
+const maxCells = 10;
+
+appendRow.onclick = function () {
```

Comment:
It's recommended to use arrow functions instead of the 'function' keyword. Arrow functions are more concise and fix some quirks of the 'this' keyword.